woocommerce/woocommerce-android
# Shipping labels M3 tracks

This PR adds event tracking to shipping label creation flow. The following events are tracked:

- The main flow steps completed (action events)
- Address validation results (action events)
- Button taps (view events)
- Spinner taps (view events)

**To test:**
1. Select an order eligible for SL creation
2. Go through the purchase flow
3. Verify each step is tracked (**shipping_label_purchase_flow**: `state` = `[started, origin_address_complete, destination_address_complete, packages_selected, carrier_rates_selected, customs_complete, purchase_initiated, purchase_succeeded, purchase_failed]`
4. Verify the purchase `amount` is tracked with the `purchase_initiated` step

~Note: This PR is a draft until the purchase functionality is implemented.~
